### PR TITLE
MD-123: Implementar edição de produtos

### PR DESCRIPTION
**Pull request by EricksonSiqueira**

### CODE REVIEW

File: src/components/EditableFoodCard/index.tsx [hunk 1]
```diff
@@ -1,59 +1,61 @@
...
           <div className={styles.editableFoodCardTextWrapper}>
-            <h3 className={styles.editableFoodCardTitle}>{title}</h3>
-            <p className={styles.editableFoodCardDescription}>{description}</p>
+            <h3 className={styles.editableFoodCardTitle}>{product.name}</h3>
+            <p className={styles.editableFoodCardDescription}>
+              {product.description}
+            </p>
           </div>
-          <Prices price={price} discountedPrice={discountedPrice} />
+          <Prices price={+product.price} discountedPrice={product.discount} />
```
Comment (claudiocassimiro) [inline src/components/EditableFoodCard/index.tsx:40]:
```suggestion
          <Prices price={+product.price} discountedPrice={product.discount} />
```
what is this + before product.price?

File: src/components/Forms/EditProductForm/index.tsx [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+  const toBase64 = (file: File | undefined) =>
+    new Promise((resolve, reject) => {
+      // vai sair daqui
+      if (file) {
+        const reader = new FileReader();
+        reader.readAsDataURL(file);
+        reader.onload = () => resolve(reader.result);
+        reader.onerror = reject;
+      }
+    });
```
Comment (claudiocassimiro) [inline src/components/Forms/EditProductForm/index.tsx:121]: I think thats Ton already create this function on utils file, and maybe you can use here. The function name is convertFileToBase64, if works you can change too on CreateProductForm

File: src/components/EditableFoodCard/index.tsx (hunk 1 above)
Comment (EricksonSiqueira) [reply src/components/EditableFoodCard/index.tsx:40]: is the same as `Number()`, it is converting string to number

Comment (claudiocassimiro) [reply src/components/EditableFoodCard/index.tsx:40]: It's cool

**Review by claudiocassimiro — APPROVED** [review]

**Review by ebertonArouxa — APPROVED** [review]

**Pull request merged**